docs/UserGuide.md

```diff
@@ -66,24 +67,25 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Adding a student: `add`
 
-### Adding a person: `add`
-
-Adds a person to the address book.
+Adds a student to Reeve.
```

Comment:
You may want to add a `(written by: AUTHOR)` statement before the part that you wrote because I think they want to see who wrote what.

Reply:
Right! Thanks for pointing it out! I have made the necessary changes.